files/es/web/javascript/index.md

```diff
@@ -0,0 +1,119 @@
+---
+title: JavaScript
+slug: Web/JavaScript
+tags:
+ - Aprender
+ - JavaScript
+ - 'l10n:priority'
+translation_of: Web/JavaScript
+---
+{{JsSidebar}}
+
+**JavaScript** (**JS**) es un lenguaje de programación ligero, interpretado, o compilado [justo-a-tiempo](https://en.wikipedia.org/wiki/Just-in-time_compilation) (just-in-time) con {{Glossary("First-class Function", "funciones de primera clase")}}. Si bien es más conocido como un lenguaje de scripting (secuencias de comandos) para páginas web, y es usado en [muchos entornos fuera del navegador](https://en.wikipedia.org/wiki/JavaScript#Uses_outside_Web_pages), tal como {{Glossary("Node.js")}}, [Apache CouchDB](https://couchdb.apache.org/) y [Adobe Acrobat](https://www.adobe.com/devnet/acrobat/javascript.html) JavaScript es un lenguaje de {{Glossary ("Prototype-based programming", "programación basada en prototipos")}}, multiparadigma, de un solo hilo, dinámico, con soporte para programación orientada a objetos, imperativa y declarativa (por ejemplo programación funcional). Lee más en [acerca de JavaScript](/es/docs/Web/JavaScript/About_JavaScript).
+
+Esta sección está dedicada al lenguaje JavaScript en sí, y no a las partes que son específicas de las páginas web u otros entornos host. Para información acerca de {{Glossary("API","APIs")}} específicas para páginas Web, consulta [APIs Web](/es/docs/Web/API ) y {{Glossary("DOM")}}.
+
+El estándar para JavaScript es [ECMAScript](/es/docs/Web/JavaScript/Language_Resources) (ECMA-262) y [la especificación de la API para la Internacionalización de ECMAScript](https://tc39.es/ecma402/) (ECMA-402). La documentación en MDN está basada enteramente en las últimas versiones preliminares de ECMA-262 y ECMA-402. Y en algunos casos donde algunas [propuestas para nuevas funciones para ECMAScript](https://github.com/tc39/proposals) ya hayan sido implementadas en los navegadores, la documentación y algunos artículos de MDN pueden hacer uso de algunas de estas funciones.
+
+No confundas JavaScript con el [lenguaje de programación Java](https://en.wikipedia.org/wiki/Java_(programming_language) ). Ambos "Java" y "JavaScript" son marcas o marcas registradas de Oracle en los Estados Unidos y otros países, Sin embargo, los dos lenguajes de programación tienen sintaxis, semántica y usos muy diferentes.
+
+> **Callout:** **Quieres transformarte en un desarrollador web front-end?**
+>
+> Hemos reunido un curso que incluye toda la información esencial que necesitas para trabajar hacia tu objetivo.
+> 
+> [Empieza aquí](es/docs/Learn/Front-end_web_developer)
+> 
+
+## Tutoriales
+
+Aprende a programar en JavaScript con guías y tutoriales.
+
+### Para completos principiantes
+
+Dirígete a nuestro tema del [Area de Aprendizaje JavaScript](/es/docs/Learn/JavaScript ) si quieres aprender JavaScript pero no tienes experiencia previa de JavaScript o programación. Los módulos completos disponibles allí son los siguientes:
```

Comment:
Being a bit picky here, but there's an extra space after `(/es/docs/Learn/JavaScript )` (in the link)

Reply:
good eye! :+1: